core/src/main/resources/static/js/requestTopics.js

```diff
@@ -73,6 +74,9 @@ app.controller("requestTopicsCtrl", function($scope, $http, $location, $window)
                                 $scope.requestTitle = "Topic Promotion Request"
                                 $scope.addTopic.envNameToDisplay = envName;
                                 $scope.addTopic.description = "Topic Promotion";
+                            }else if(reqType != null && reqType === 'CreateTopic'){
+                            $scope.addTopic.topicpartitions = $scope.envTopicMap.defaultPartitions  + " (default)"
+                            $scope.addTopic.replicationfactor = $scope.envTopicMap.defaultRepFactor + " (default)"
```

Comment:
```suggestion
                            $scope.addTopic.replicationfactor = $scope.envTopicMap.defaultRepFactor + " (default)";
```
probably js is ok without eol characters